### DISCUSSION TITLE
高级操作怎么玩啊，尤其是最后几个

### DISCUSSION THREAD

**Discussion by wg961** [GaiZhenbiao/ChuanhuChatGPT#768] [Q&A] [answered]
纯小白玩了几个小时实在是没玩透，尤其是最后的“用户名”摸不着头脑，大佬救命啊
![QQ截图20230520224352](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/131155116/434ab85b-3422-46b5-990a-354443cf078e)
![QQ截图20230520220910](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/131155116/c70f0845-fb1e-47b8-8d2c-3dc93d9568fa)

**Comment by Keldos-Li** [comment, marked as answer]
高级参数请参考该页：https://platform.openai.com/docs/api-reference/chat/create

关于用户名：https://platform.openai.com/docs/guides/safety-best-practices/end-user-ids
